public/components/common/modules/panel/components/module-side-panel.tsx

```diff
@@ -22,8 +22,8 @@ export const ModuleSidePanel = ({ navIsDocked = false, children, ...props }) =>
     <EuiCollapsibleNav
       isOpen={navIsOpen}
       isDocked={navIsDocked}
-      showCloseButton={true}
-      maskProps={{ headerZindexLocation: 'below', className: 'wz-no-display' }}
+      ownFocus={false}
```

Comment:
**thought**: I tested without this property definition, and add a mask under the component. It looks better visually. I checked in a previous version `4.3.10` and this doesn't have the mask, so I guess we could accept this change.